bijection-scrooge/src/main/scala/com/twitter/bijection/scrooge/ScroogeCodecs.scala

```diff
@@ -35,10 +35,19 @@ object BinaryScalaCodec {
 }
 
 class BinaryScalaCodec[T <: ThriftStruct](c: ThriftStructCodec[T])
```

Comment:
Yeah sorry I did a got commit -am and missed the files. :/

On Wednesday, June 3, 2015, P. Oscar Boykin <notifications@github.com>
wrote:

> In
> bijection-scrooge/src/main/scala/com/twitter/bijection/scrooge/ScroogeCodecs.scala
> <https://github.com/twitter/bijection/pull/221#discussion_r31660600>:
>
> > @@ -35,10 +35,19 @@ object BinaryScalaCodec {
> >  }
> >
> >  class BinaryScalaCodec[T <: ThriftStruct](c: ThriftStructCodec[T])
>
> Somehow I missed the new protocol implementation (was that left out of the
> first commit?) I get it now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/bijection/pull/221/files#r31660600>.
>